installer/pkg/config-generator/tls.go

```diff
@@ -239,18 +239,18 @@ func (c *ConfigGenerator) GenerateTLSConfig(clusterDir string) error {
 		return fmt.Errorf("failed to generate kubelet certificate: %v", err)
 	}
 
-	// TNC certs
-	tncDomain := fmt.Sprintf("%s-tnc.%s", c.Name, c.BaseDomain)
+	// MachineConfigServer certs
+	mcsDomain := fmt.Sprintf("%s-tnc.%s", c.Name, c.BaseDomain)
```

Comment:
Do we want to update the template to `%s-mcs.%s` or similar?  And you probably want to make similar changes to `pkg/asset/tls`.

Reply:
+1

Reply:
https://jira.coreos.com/browse/CORS-833 is going to consolidate the api endpoints. So left it for now.